common/src/java/org/apache/hive/http/ProfileServlet.java

```diff
@@ -45,19 +47,19 @@
  * //  -m method         fully qualified method name: 'ClassName.methodName'
  * //  -t                profile different threads separately
  * //  -s                simple class names instead of FQN
- * //  -o fmt[,fmt...]   output format: summary|traces|flat|collapsed|svg|tree|jfr
+ * //  -o fmt[,fmt...]   output format: summary|traces|flat|collapsed|svg|tree|jfr|html
```

Comment:
should we remove `svg` if that is not supported anymore?

Reply:
it's supported in older versions of the profiler, so the user might want to use that